templates/csharp/app-mvvm/ViewLocator.cs

```diff
@@ -8,19 +8,17 @@ namespace AvaloniaAppTemplate;
 public class ViewLocator : IDataTemplate
 {
 
-    public Control? Build(object? data)
+    public Control? Build(object? param)
     {
-        if (data is null)
+        if (param is null)
             return null;
         
-        var name = data.GetType().FullName!.Replace("ViewModel", "View", StringComparison.Ordinal);
+        var name = param.GetType().FullName!.Replace("ViewModel", "View", StringComparison.Ordinal);
         var type = Type.GetType(name);
 
         if (type != null)
         {
-            var control = (Control)Activator.CreateInstance(type)!;
-            control.DataContext = data;
-            return control;
+            return (Control)Activator.CreateInstance(type)!;
         }
```

Comment:
Hey, is there a reason for making this change? The F# template still has the previous default behavior (which I personally prefer)

Reply:
This change was made to match our other template: https://github.com/AvaloniaUI/avalonia-dotnet-templates/blob/main/templates/csharp/xplat/AvaloniaTest/ViewLocator.cs

Also, hardcoding DataContext in this code potentially can be problematic. Typically, DataContext should be inherited from parent instead.

Reply:
I didn't change F# template, as I can't confidently code in F# or test it. 
If you think it should also be updated in any way, and can test it, please open a PR, I can review it at least.

Reply:
Yeah, this makes sense, thanks for getting back to me. I was so happy about being a contributor with #238 but I understand your reasoning.